packages/amplify-category-api/src/commands/api/add-graphql-datasource.ts

```diff
@@ -233,12 +236,12 @@ export const readSchema = (graphqlSchemaFilePath: string): graphql.DocumentNode
     currentGraphQLSchemaDoc = graphql.parse(graphqlSchemaRaw);
   } catch (err) {
     const relativePathToInput = path.relative(process.cwd(), graphqlSchemaRaw);
-
-    const error = new Error(`Could not parse graphql schema \n${relativePathToInput}\n${err.message}`);
-
-    error.stack = undefined;
-
-    throw error;
+    // can be changed to GraphQlError when the error type "GraphQlError"  gets updated in amplify-cli-core
+    throw new AmplifyError('UserInputError', {
+      message: `Could not parse graphql scehma \n${relativePathToInput}\n`,
+      details: err.message,
+      link: 'https://docs.amplify.aws/cli-legacy/graphql-transformer/relational/',
```

Comment:
Is this doc relevant here? Looks like we're pointing to `legacy` docs.

Reply:
I couldn't find RDS docs on our new amplify docs site